app/queries/get_content_item_ids_with_fallbacks.rb

```diff
@@ -0,0 +1,55 @@
+module Queries
+  class GetContentItemIdsWithFallbacks
+    extend ArelHelpers
+
+    class << self
```

Comment:
I'm using it here so I can make `order_by_clause` private. I could use `private_class_method` instead I suppose?